### DISCUSSION TITLE
Welcome to TorrentHunt Discussions!

### DISCUSSION THREAD

**Discussion by hemantapkh** [hemantapkh/TorrentHunt#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bacchilega** [comment]
I don't know if you know Bit Che. 
It's a very old search engine that allows you to search for torrents online. 
I would like to create a search engine like that but with updated links.
Unfortuntely is not open-source.
I have collected a couple of sites that could be added.

#The Pirate Bay https://pirate-bay-proxy.org/it/
#RARBG https://rarbg.to/torrents.php
#1337X https://1337xto.to/
#YTS
#torrentfunk2 https://www.torrentfunk2.com/
#EZTV
#Zooqle
#TorrentDownloads #https://www.torrentdownloads.pro/
#Lime Torrents
#torlock https://www.torlock.com/
#Il Corsaro Nero https://www.ilcorsaronero.in/
#ExtraTorrent
#LimeTorrents
#https://idope.se/
#https://www.skidrowreloaded.com/
#solidtorrents.to
#https://katcr.to/
#www.divxtotal.re
#gtdb https://www.gtdb.to/home.php
#https://www.torrentmac.net/
#https://mactorrents.io/
#https://www.elitetorrent.nz/
#bittorent bittorrent.am
#isohunts https://isohunts.to/
